templates/patterns/carousel/carousel.ui_patterns.yml

```diff
@@ -1,77 +1,61 @@
 carousel:
-  label: "Carousel"
-  description: "A slideshow component for cycling through elements, like a carousel. https://v5.getbootstrap.com/docs/5.0/components/carousel/"
+  label: 'Carousel'
+  description: 'A slideshow component for cycling through elements, like a carousel. For more information: https://v5.getbootstrap.com/docs/5.0/components/carousel/'
   settings:
     fade:
       type: boolean
-      label: "Fade"
-      description: "Animate slides with a fade transition instead of a slide."
+      label: 'Fade'
+      description: 'Animate slides with a fade transition instead of a slide. False by default.'
       preview: true
     with_controls:
       type: boolean
-      label: "With controls"
-      description: "With controls (default: false)."
+      label: 'With controls'
+      description: 'Enable the controls (prev and next buttons). False by default.'
       preview: true
-    prev_label:
-      type: select
-      label: "Previous label"
-      description: "The previous label value."
-      options:
-        1: "Previous"
-        2: "Previous slide"
-      preview: 1
-    next_label:
-      type: select
-      label: "Next label"
-      description: "The next label value."
-      options:
-        1: "Next"
-        2: "Next slide"
-      preview: 1
     with_indicators:
       type: boolean
-      label: "With indicators"
-      description: "With indicators (default: false)."
+      label: 'With indicators'
+      description: 'With indicators (default: false).'
       preview: true
     autoplay:
       type: boolean
-      label: "Disable autoplay"
-      description: "Disable auto slide movement."
+      label: 'Disable autoplay'
+      description: 'True by default. If set to false the carousel will not slide automatically.'
       preview: true
```

Comment:
This can be deleted, the default value is 'true'. Check if also set at markup rendering test, then there can be eliminated.